ui/lib/components/accordion/accordion.dart

```diff
@@ -0,0 +1,150 @@
+import 'dart:html';
+
+/*
+<div class="accordion">
+  <div class="accordion-item">
+    <div class="accordion-header__wrapper">
```

Comment:
nit - what do you think about this name?
```suggestion
    <div class="accordion-item__header">
```
and similarly for the body wrapper below - `accordion-item__body`

Reply:
(and I think you can keep the `_headerWrapper` name below, this is just a comment for the css class)